internal/middleware/util.go

```diff
@@ -85,66 +85,42 @@ func ExtractStringField(data map[string]interface{}, key string) (string, error)
 }
 
 // ValidateCommonResourceData Validates the "commonResourceData" field using a predefined schema.
-func ValidateCommonResourceData(resourceType string, resource map[string]interface{}) error {
-
+func ValidateCommonData(resourceType string, commonData map[string]interface{}) error {
```

Comment:
suggest `ValidateCommonRepresentation`

Anywhere we were generically saying "data" we should start saying "representation" to be specific about _what_ data we're talking about.

Reply:
Done.